src/main/resources/org/jfrog/hudson/ArtifactoryBuilder/global.jelly

```diff
@@ -2,21 +2,22 @@
          xmlns:f="/lib/form"
          xmlns:st="jelly:stapler"
          xmlns:r="/lib/jfrog"
-         xmlns:c="/lib/credentials">
+         xmlns:c="/lib/credentials" xmlns:l="/lib/layout">
     <st:adjunct includes="lib.jfrog.repos.credentials"/>
     <f:section title="Artifactory">
 
         <f:entry
             help="/plugin/artifactory/help/ArtifactoryBuilder/help-useCredentialsPlugin.html">
-                <input id="useCredentialsPlugin" type="checkbox" onclick="updateViewForCredentialsMethod(this.checked)"
-                   name="useCredentialsPlugin" value="${descriptor.useCredentialsPlugin}"
-                        field="useCredentialsPlugin">Use the Credentials Plugin
-                </input>
+                <f:checkbox id="useCredentialsPlugin"
```

Comment:
This isn't strictly related but it's an accessibility bug I noticed while testing out the UI